src/plugins/action-queue/ActionQueueStore.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+
+import { asMaybe } from 'cleaners'
+import { type EdgeAccount } from 'edge-core-js'
+
+import { asActionProgram, asActionProgramState } from './cleaners'
+import { type ActionProgram, type ActionProgramState, type ActionQueueMap } from './types'
+
+export const ACTION_QUEUE_DATASTORE_ID = 'actionQueue'
+
+export type ActionQueueStore = {
+  set: (itemId: string, data: any) => Promise<void>,
+  get: (itemId: string) => Promise<any>,
```

Comment:
Maybe use `mixed` instead of `any`, since that will error if we forget to use a cleaner / uncleaner.